accounts-db/src/accounts_db.rs

```diff
@@ -1113,7 +1113,6 @@ impl AccountStorageEntry {
             return None;
         }
 
-        APPEND_VEC_REOPEN_AS_FILE_IO.fetch_add(1, Ordering::Relaxed);
```

Comment:
Why is this removed?

Reply:
https://github.com/anza-xyz/agave/blob/53e839c5915c8ba9bc86de3b6a89c389c3f3e3bf/accounts-db/src/accounts_db.rs#L4225
https://github.com/anza-xyz/agave/blob/53e839c5915c8ba9bc86de3b6a89c389c3f3e3bf/accounts-db/src/accounts_db.rs#L5700
https://github.com/anza-xyz/agave/blob/53e839c5915c8ba9bc86de3b6a89c389c3f3e3bf/accounts-db/src/accounts_db.rs#L1088
https://github.com/anza-xyz/agave/blob/53e839c5915c8ba9bc86de3b6a89c389c3f3e3bf/accounts-db/src/accounts_file.rs#L316

Reply:
It seems for reopen_as_readonly(), the store was created with `AppendVec::new`, not `new_from_file_unchecked()`. 
When this store is created, only *APPEND_VEC_MMAPPED_FILES_OPEN* increased.

Reply:
I see. You are right. APPEND_VEC_REOPEN_AS_FILE_IO would be correctly increased by 1 with the PR. 
What about  `APPEND_VEC_MMAPPED_FILES_OPEN`?
It looks like we are double increase `APPEND_VEC_MMAPPED_FILES_OPEN`. One from AppendVec::new(), and One from new_from_file(). 

Should we decrease ` APPEND_VEC_MMAPPED_FILES_OPEN.fetch_sub(1, Ordering::Relaxed);` here?

Reply:
yeah. you are right!